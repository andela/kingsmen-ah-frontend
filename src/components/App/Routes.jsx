import React from 'react';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
import Home from '@components/views/Home';
import Login from '@components/views/Login';
import NotFound from '@components/views/NotFound';
import Profile from '@components/views/Profile/Profile';

const Routes = () => (
  <Router>
    <Switch>
      <Route exact path="/" component={Home} />
      <Route exact path="/login" component={Login} />
      <Route exact path="/profile" component={Profile} />
      <Route path="/" component={NotFound} />
    </Switch>
  </Router>
);

export default Routes;
