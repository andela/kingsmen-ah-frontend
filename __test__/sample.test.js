import React from 'react';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import App from '@components/App';

configure({ adapter: new Adapter() });

describe('SAMPLE TEST', () => {
  it('should return a successful sample test ', () => {
    expect(true).toBeTruthy();
  });
});

describe('App', () => {
  it('should render the App component correctly', () => {
    const shallowWrapper = shallow(<App />);

    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
