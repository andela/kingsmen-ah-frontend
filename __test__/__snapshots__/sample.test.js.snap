// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App component correctly 1`] = `<Routes />`;
